### ISSUE TITLE
FzfPreviewProjectGrep very slow after migrating from ruby to python

### ISSUE THREAD

**Issue by wcwagner** [yuki-yano/fzf-preview.vim#53]
Hey, I just started using this extension over fzf.vim and it has been great so far! Although, I recently upgraded via vim-plug, and noticed that `:FzfPreviewProjectGrep` was unbearably slow on a decently sized project.

After looking a little futher, I noticed https://github.com/yuki-ycino/fzf-preview.vim/commit/fd8f09dc7c9c997e4ed9cf5b13f1e0b72b758aae switched the `preview_fzf_grep` scrip to use python3 instead of ruby. After downgrading to the commit right before that, the latency is back to normal.

I'm not exactly sure why it's slower, but I'll try and dig a little more. Anyways, thanks for the great extension, I'm sure we'll be able to figure this one out.

**Comment by geodimm** [comment]
Hi @wcwagner ,

I did a little bit of digging myself and what I found out is that it's not the change from `ruby` to `python` that makes that slow, it's the usage of devicons.

The command is much, much faster with `let g:fzf_preview_use_dev_icons = 0`.

For context, I'm testing against a repository with over 6000 files (130,000 lines) and the `python` scripts runs in 50ms, the rest of the time is spent adding devicons.

Probably the commit that actually introduces this issue is https://github.com/yuki-ycino/fzf-preview.vim/commit/cde39058463dfa26fa42ffe1cd823f360dfc51ed (which is actually the very next one and that's why rolling back to https://github.com/yuki-ycino/fzf-preview.vim/commit/126710c41c136ce5538cd96afd05f32f6e8a7f20 worked for you)
